### PR TITLE
Added meet-up caption

### LINKED ISSUE #2734: [About] Open Source Africa Community: Add caption to "Community First" picture
**Desired Behavior**
Add caption to Layer5 Community Members picture on https://layer5.io/company/about page.

"_Layer5 community members at Open Source Africa Community conference, March 2022._"

**Screenshots / Mockups**
Screen Shot 2022-04-27 at 2 05 47 PM

---

<h4>Contributor Resources</h4>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by St80ene**
Signed-off-by: etiene <etienejames5@gmail.com>

**Description**

This PR fixes #2734

**Notes for Reviewers**
- Added caption to Open Source Community Meetup, March 2022 picture
### Screenshot
Screenshot from 2022-04-28 17-35-29

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Thank you, @St80ene

**Comment by leecalcote** [comment]
Is the site building and running for you locally? The build preview failed.

**Comment by St80ene** [comment]
It required installing a package called "react-is" during the development.
I removed it before pushing because it was already a part of the packages.
I'll have to add it back. I wanted to avoid having conflicts.

File: src/sections/Company/About/index.js [hunk 1]
```diff
@@ -60,6 +60,7 @@ const About = () => {
               <Col xs={12} sm={6}>
                 <div className="head-images">
                   <img src={Community_meetup} alt="Community_meetup" />
+                  <p className="meetup-caption">Layer5 community members at Open Source Africa Community conference, March 2022.</p>
```
Comment (Nikhil-Ladha) [inline src/sections/Company/About/index.js:63]: Insteead of `p` tga use `figcaption` and wrap the complete `img` and `figcaption` within a `figure`

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
The extra package is redundant and not required.
Also, please undo the changes to the file which are not related to the issue.

**Comment by leecalcote** [comment]
Merge conflicts...

**Comment by leecalcote** [comment]
Gents, allow me to collaborate here and get this over the hump....

**Comment by St80ene** [comment]
No problem. I'll do that. I've been stuck on that same issue for some time now.
Thanks

**Pull request closed without merging**